inputfiles/sherpa/LHC-NLO-ZplusJet/Run.dat

```diff
@@ -26,5 +30,4 @@
 
 (selector){
   Mass 13 -13 60. 120.;
-  NJetFinder 1 20. 0. 0.7 -1;
```

Comment:
Same here, why is the JetFinder not needed?

Reply:
See above